client-runtime/src/test/java/com/microsoft/rest/v2/RestProxyTests.java

```diff
@@ -1010,14 +1009,14 @@ public void service19PutWithHeaderApplicationJsonContentTypeAndCharsetAndStringB
     public void service19PutWithHeaderApplicationJsonContentTypeAndCharsetAndStringBodyWithEmptyBody() {
         final HttpBinJSON result = createService(Service19.class)
                 .putWithHeaderApplicationJsonContentTypeAndCharsetAndStringBody("");
-        assertEquals("\"\"", result.data);
+        assertEquals("", result.data);
     }
 
     @Test
     public void service19PutWithHeaderApplicationJsonContentTypeAndCharsetAndStringBodyWithNonEmptyBody() {
         final HttpBinJSON result = createService(Service19.class)
                 .putWithHeaderApplicationJsonContentTypeAndCharsetAndStringBody("soups and stuff");
-        assertEquals("\"soups and stuff\"", result.data);
+        assertEquals("soups and stuff", result.data);
```

Comment:
This is because RestProxy is no longer serializing the String before putting it in the request body?